tests/_runner.lua

```diff
@@ -43,7 +43,54 @@ runner.step_kill_clients = function(step)
     end
 end
 
-runner.run_steps = function(steps, options)
+--- Print a message if verbose mode is enabled
+-- @tparam string message The message to print
+function runner.verbose(message)
+    if verbose then
+        io.stderr:write(message .. "\n")
+    end
+end
+
+--- When using run_direct(), this function indicates that the test is now done.
+-- @tparam[opt=nil] string message An error message explaining test failure.
```

Comment:
```suggestion
-- @tparam[opt=nil] string message An error message explaining the test failure, if it failed.
```
It was not clear to me if this was called always, or only for errors.